src/eureka/lib/centerdriver.py

```diff
@@ -97,8 +97,8 @@ def centerdriver(method, data, guess, trim, radius, size, i, m, meta,
         extra = sy, sx  # Gaussian 1-sigma half-widths
 
     # only plot when we do the second fit
-    if (meta.isplots_S3 >= 5 and method[-4:] == '_sec' and i < meta.nplots and
-            len(img) == (2 * meta.ctr_cutout_size + 1)):
+    if (meta.isplots_S3 >= 5 and method[-4:] == '_sec' and
+            i < meta.nplots):
```

Comment:
No need to put that `i < meta.nplots):` on a new line - you can just move that back up to the line above (I think I may have accidentally told you to do that during our meeting today)